images/installer/Dockerfile

```diff
@@ -8,7 +8,9 @@ USER root
 COPY images/installer/origin-extra-root /
 
 # install ansible and deps
-RUN INSTALL_PKGS="python-lxml python-dns pyOpenSSL python2-cryptography openssl python2-passlib httpd-tools openssh-clients origin-clients iproute patch" \
+RUN REPO_PKGS="http://resources.ovirt.org/pub/yum-repo/ovirt-release42.rpm" \
```

Comment:
Is there a precedent for a platform adding a specialized yum repo for their api libraries?

For Dockerfile.rhel7 - how would a rhel customer consume these ovirt packages?  Would they have to subscribe to a RHV channel?

I see below that epel has the azure cli libraries.  I don't see anything about openstack, gce, or other platforms.

Reply:
What we would need is content sharing so the ovirt sdk rpm can be installed inside the container and then we would be able to use it as is.